viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -824,7 +824,7 @@ extern int viam_carto_lib_init(viam_carto_lib **ppVCL, int minloglevel,
         return VIAM_CARTO_OUT_OF_MEMORY;
     }
     google::InitGoogleLogging("cartographer");
-    FLAGS_logtostderr = 1;
+    FLAGS_logtostdout = 1;
```

Comment:
Can't believe we didn't figure this out sooner :'D 

Great catch!!!

Reply:
1. Great find!
2. I'd be sure you check the lowest common version of glog we build cartographer with across all our platforms to be sure this will work given this feature was only added `Feb 19, 2022`: https://github.com/google/glog/commit/180b700ddc99f8f04ccfdb5aab125f493ed891a8 . It looks like the first pinned version that has the feature is https://github.com/google/glog/releases/tag/v0.6.0 so I'd be sure we are at least using that version.